src/test/alert_tests.cpp

```diff
@@ -8,7 +8,9 @@
 #include "chain.h"
 #include "chainparams.h"
 #include "clientversion.h"
+namespace alert_tests {
```

Comment:
Agree. It's this way because I aligned it with https://github.com/bitcoin/bitcoin/blob/master/src/bench/checkblock.cpp#L12
But I'll remove the namespace now.